### PR TITLE
Implement front end UI for Admin page

### PR DESCRIPTION
**Pull request by AnkitLakkapragada**
### Summary 

This pull request is the first step towards implementing the admin page that approves and rejects reviewers for apartments.

The page can be accessed at ` /admincuapts1234`. The next steps include creating an API to fetch all pending reviews and giving this button functionality and this webpage security.  

### Test Plan 

image

### Breaking Changes  

 
None

### CODE REVIEW

**Review by cyrus14 — DISMISSED** [review]
Loving this front end UI for Admin Page. I think this is a great foundation for the Admin Page. Once the backend is figured out I think some adjustments will need to be made to have the approve and reject be placed in a way that is aligned with the review that you are deciding for. For now though, this is great for the set up!

File: frontend/src/components/utils/NavBar/index.tsx [hunk 1]
```diff
@@ -270,6 +305,9 @@ const NavBar = ({ headersData, user, setUser }: Props): ReactElement => {
         <Grid item md={6} className={searchBar ? search : searchHidden}>
           {auto()}
         </Grid>
+        <Grid item md={1} className={menu} container justifyContent="flex-end">
```
Comment (thuypham03) [inline frontend/src/components/utils/NavBar/index.tsx:308]: The logic will be better if you have another boolean isAdmin then you do {isAdmin && <Button>}, the button should only render the button, not to check whether it's admin or not

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -90,6 +90,19 @@ hotjar.initialize(HJID, HJSV);
 
 const App = (): ReactElement => {
   const [user, setUser] = useState<firebase.User | null>(null);
+  const admins: string[] = [
```
Comment (thuypham03) [inline frontend/src/App.tsx:93]: This should be moved to const file.

**Review by AnkitLakkapragada — COMMENTED** [review]
Looks good, this change is really needed for the front-end UI and also the first step for the admin page that will allow us to approve reviews.

**Review by thuypham03 — APPROVED** [review]
This PR finish implementation for admin page UI. In the future, the button need to be functional to change review from PENDING to ACCPETED/ DECLINED!

**Pull request merged**